setupext/platform.py

```diff
@@ -49,22 +49,15 @@
     platform_specific['extra_link_args'] = ['/DEBUG']
```

Comment:
So Windows will have debug symbols, while OSX and Linux don't?

Reply:
This is an odd one.  For some reason (I can't explain) the linux binary is bloated to 10 times all the other ones.  I didn't want to have to strip because stripping will get rid of way to much.   I think that -g3 was an accident on my part many versions back (I think I did the edit because I needed the symbols).  You can see from setupext that we deliberately override the -g0 to -g3 for tracing and coverage builds.   

So what is the correct compiler flags for a release?

Reply:
I could also be a leftover from ancient times, when I actively fiddled around with the code. If we omit debug flags completely, we are not able to retrieve proper stacktraces from our users. I saw how you instructed a user to get a nice tracing; pinning down the issue. So I guess we can safely omit the symbols, and instruct people to reproduce the found crashes with a debug build.

Reply:
Okay let me make it ``-g`` so we have some symbols.   The size reduction didn't work anyway so there is likely some other bug that I will fix later.

Reply:
It seems like ggdb is added by distutils (it derives the CFLAGS from the ones used to compile python itself). This flag overrides g0 and the binary is huge after all.